tweepy/api.py

```diff
@@ -1383,3 +1481,114 @@ def _pack_image(filename, max_size, form_field="image", f=None):
         }
 
         return headers, body
+
+    @staticmethod
+    def _chunk_media(command, filename, max_size, form_field="media", chunk_size=4096, f=None, media_id=None, segment_index=0, is_direct_message=False):
+        fp = None
+        if command == 'init':
+            if f is None:
+                file_size = os.path.getsize(filename)
+                try:
+                    if file_size > (max_size * 1024):
+                        raise TweepError('File is too big, must be less than %skb.' % max_size)
+                except os.error as e:
+                    raise TweepError('Unable to access file: %s' % e.strerror)
```

Comment:
`os.path.getsize` is outside the `try` block and nothing else in it will raise `OSError`.